### DISCUSSION TITLE
Student Benefits not available

### DISCUSSION THREAD

**Discussion by yashnahar01** [community/community#130046] [GitHub Education]
### Hi everyone,

**Hi GiHub Team, My GitHub Education request was approved 3 days ago , but I have not yet received my benefits pack. Can you kindly look into it?**
<img width="335" alt="image" src="https://github.com/community/community/assets/160831458/004fb6ce-1af2-4663-b0e6-60583c094124">

**Comment by davevad93** [comment]
Hi @yashnahar01 , 

Take a look at this discussion #111352 which states the following:

- ***We aim to provide you with your educational benefits within 3 days of this approval status. However, during busy periods like the start of a new semester, it may take a bit longer.***

  ***If you have not received your education benefits within 5 days of your approval status please raise a discussion on the [GitHub Education community](https://github.com/orgs/community/discussions/categories/github-education) and the Education community team will be able to assist in resolving this for you. 🌟***

Sometimes it can take even longer, like a week or more, anyway if after 5 days you still don't have your benefits, someone from GitHub's education staff will surely assist you.
